data/avro-schema/src/main/resources/avro/net/corda/data/crypto/wire/ops/flow/queries/ByIdsFlowQuery.avsc

```diff
@@ -2,14 +2,14 @@
   "type": "record",
   "name": "ByIdsFlowQuery",
   "namespace": "net.corda.data.crypto.wire.ops.flow.queries",
-  "doc": "Request to lookup for keys by their ids",
+  "doc": "Request to lookup for keys by their ids. Ids can be either short or full key ids. The maximum number of items is 20",
   "fields": [
     {
       "name": "keyIds",
-      "type": {
-        "type": "array",
-        "items": "string"
-      },
+      "type": [
+        "net.corda.data.crypto.ShortHashes",
```

Comment:
We should not allow 48 bit ShortHashes for this query; it'd make it pointlessly easier for flows to pick up keypairs they shouldn't.

Reply:
Removed short key ids from `ByIdsFlowQuery`.